core/templates/dev/head/pages/base.html

```diff
@@ -136,6 +136,7 @@
     {% if iframed %}
       {{ warnings_and_loader() }}
     {% else %}
+    <div role="button" tabindex="0" ng-click="utilsService.skipTo($event)" id="skiptocontent" data-link="blk-content">Skip to Main Content</div>
```

Comment:
@anthkris I also just noticed that all of the elements below this skip content button are indented once... I have it fixed on my branch now.